decidim-assemblies/lib/decidim/assemblies/menu.rb

```diff
@@ -70,6 +70,8 @@ def self.register_admin_assemblies_attachments_menu!
       def self.register_admin_assemblies_components_menu!
         Decidim.menu :admin_assemblies_components_menu do |menu|
           current_participatory_space.components.each do |component|
+            next if component.trashed?
```

Comment:
Ok, but I think you could have used the above line like : 

`current_participatory_space.components.not_trashed.each do |component|`